### PR TITLE
Create Solid Labs

### PR DESCRIPTION
**Pull request by Mitzi-Laszlo**
A list of research institutions working on Solid.

### CODE REVIEW

**Comment by RubenVerborgh** [comment]
Why a new PR, did something go wrong with the previous? (It's important to have that context, and a link, etc.)

**Comment by Mitzi-Laszlo** [comment]
Tim's review meant that GitHub blocked the merge until only Tim re-reviewed it despite the changes already being made.

**Comment by RubenVerborgh** [comment]
Still need a link back though. This seems to be a new PR.

**Comment by RubenVerborgh** [comment]
Original is https://github.com/solid/solidproject.org/pull/244.

Next time, we should aim for repairing the PR, so we don't lose history. I can't see the changes between #268 and #244 now, for instance.

File: solid-labs.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+layout: page
+title: "Labs"
+permalink: labs
+---
+
+Here are some academic groups specifically doing research around Sold. If you are interested in being listed send an email to info@solidproject.org with a description of the research project and link to the website.
```
Comment (RubenVerborgh) [inline solid-labs.md:7]:
```suggestion
Several academic groups conduct research around Solid. If you are interested in being listed send an email to info@solidproject.org with a description of the research project and link to the website. 
```

File: solid-labs.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+---
+
+Here are some academic groups specifically doing research around Sold. If you are interested in being listed send an email to info@solidproject.org with a description of the research project and link to the website. 
+
+### [MIT](https://www.mit.edu) - [CSAIL DIG](http://dig.csail.mit.edu) 
+Boston, USA 
+
+Solid started at the [CSAIL Solid Social Linked Data Lab](https://www.csail.mit.edu/research/solid-social-linked-data) in 2015, which is now [CSAIL DIG](http://dig.csail.mit.edu). The research on Solid at MIT focuses include verifiable credentials, privacy-preserving data aggregation on the decentralised web and investigating decentralised management of health and fitness data. 
+
+Contact = [Lalana Kagal](https://www.csail.mit.edu/person/lalana-kagal)
```
Comment (RubenVerborgh) [inline solid-labs.md:14]: What's up with the `=`? That is weird.

File: solid-labs.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+### [Oxford University Computer Science Lab](http://www.cs.ox.ac.uk) - [Oxford Martin School's program on Ethical Web and Data Architectures](https://www.oxfordmartin.ox.ac.uk/ethical-web-and-data-architectures). 
+
+The work at the Oxford Martin School's program on Ethical Web and Data Architectures focuses on four themes: data autonomy, data privacy, algorithmic accountability, and data sharing. In 2020, a grant was recieved to architect new ethical data systems such as Solid. 
+
+Contact = [Nigel Shadbolt](https://www.cs.ox.ac.uk/people/nigel.shadbolt/)
+
+### The [KNowledge on Web Scale (KNoWS)](https://knows.idlab.ugent.be/) group at [Ghent University](https://www.ugent.be/en) – [imec](https://www.imec-int.com/) 
+Ghent, Belgium 
+
+IMEC has been involved in Solid for several years and are currently working on the community Solid server.
```
Comment (RubenVerborgh) [inline solid-labs.md:25]:
```suggestion
Imec has been involved in Solid for several years, focusing on data interoperability, querying, scalability, and the developer experience. They are currently building the [Solid Community Server](https://github.com/solid/community-server/).
```

File: solid-labs.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+Oviedo, Spain
+
+Classes at WESO now have the tradition of a competition for the best Solid app produced during their course. 
+
+Contact = [Jose Emilio Labra Gayo](http://labra.weso.es)
+
+### [CERN](https://home.cern) - [Indico](https://indico.cern.ch/event/915906/)
+Geneva, Switzerland 
+
+CERN is heavily involved in open source, and working on making Indico Solid compatible.
```
Comment (RubenVerborgh) [inline solid-labs.md:39]: What is Indico? Or a link at least.

File: solid-labs.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+Geneva, Switzerland 
+
+CERN is heavily involved in open source, and working on making Indico Solid compatible. 
+
+Contact = [Maria Dimou](http://dimou.web.cern.ch/dimou/)
+
+### [Open University] - [Open Blockchain Group](https://blockchain.open.ac.uk/#covid-19)
+Milton Keynes, UK
+
+The Open University runs an experimental server and works on a covid-19 immunity passport Solid app.
```
Comment (RubenVerborgh) [inline solid-labs.md:46]:
```suggestion
The Open University runs an experimental server and works on a COVID-19 immunity passport Solid app. 
```

**Pull request merged**